docs/05-concepts/server-side-events.mdx

```diff
@@ -1,35 +1,45 @@
 ---
```

Comment:
**note:** This looks like there are no change in this file beside the formatting. For future occurences, please let the reviewers know that this file isn't to be reviewed if that's the case, by either mentionning it in the MR's description, or by putting it in an appropriatedly named serated commit.